client-app/pages/account/profile.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="bg-gray-100 flex-grow pt-6 pb-16 shadow-inner">
+    <div class="max-w-screen-2xl md:px-12 mx-auto">
+      <div class="flex lg:space-x-5">
+        <!-- First column-->
+        <div class="hidden lg:flex flex-col lg:w-1/5 space-y-5">
+          <AccountNavigation></AccountNavigation>
+        </div>
+
+        <!-- Second column-->
+        <div class="flex flex-col w-full lg:w-4/5 space-y-5">
+          <div class="flex justify-between items-center mx-5 lg:mx-0">
+            <h2 class="text-gray-800 text-3xl font-bold uppercase">Profile</h2>
+          </div>
+          <div class="flex flex-col bg-white shadow-sm p-6" :class="{ 'rounded border polygon-bg': !isMobile }">
+            <form @submit="onSubmit" class="lg:w-1/2 flex flex-col">
+              <!-- Errors block -->
+              <VcAlert v-if="updateProfileError" class="mb-2">There was an error during profile update</VcAlert>
+              <VcAlert v-if="!_.isEmpty(errors)" class="mb-2">
+                <strong>First name</strong> and <strong>Last name</strong> are required
+              </VcAlert>
+
+              <VcInput
+                v-model="firstName"
+                name="firstName"
+                class="mb-4"
+                label="First name"
+                placeholder="First Name"
+                is-required
+                :error-message="errors.firstName"
+              ></VcInput>
+
+              <VcInput
+                v-model="lastName"
+                name="lastName"
+                class="mb-4"
+                label="Last name"
+                placeholder="Last name"
+                is-required
+                :error-message="errors.lastName"
+              ></VcInput>
+
+              <VcInput
+                v-model="email"
+                name="email"
+                class="mb-4"
+                label="Email"
+                placeholder="Email"
+                :is-disabled="true"
```

Comment:
This can be done shortly just using `is-disabled` - if prop exists, it's `true` by default. Just like `is-required`, `is-submit`, etc.